inst/app/www/css/style.css

```diff
@@ -62,38 +87,31 @@ and prevent them fading in / out) */
 /* Tweak CSS for navpanel to make it sticky when scrolling page */
 .col-sm-3 {
   position: -webkit-sticky; /* Safari */
-  position:sticky;
-  left:0;
-  top:0;
+  position: sticky;
+  left: 0;
+  top: 0;
 }
 @media (max-width:800px){
-    .col-sm-3 {
-      position:inherit;
-}
+  .col-sm-3 {
+    position:inherit;
+  }
 }
 
-/* Tweak css for navbar styleing to prevent contrast issues*/
-.navbar-nav li a {
-  font-weight: bold; 
-  color : #000000;
+/* Tweak colours to get contrast dead on 4.5:1 */
+.navbar-default .navbar-brand {
+    color: #717465;
+    background-color: #f8f8f8;
 }
 
-.navbar-default .navbar-nav li a {
-  font-weight: bold; 
-  color : #000000;
+.navbar-default .navbar-nav>.active>a, .navbar-default .navbar-nav>.active>a:focus, .navbar-default .navbar-nav>.active>a:hover {
```

Comment:
Is this added for accessibility? I presume but want to check (curious!)

Reply:
Yup. If you have a look at the BSIL app on the page with supplementary info (Definitions maybe?) there is some greyish looking menus etc. These were throwing contrast errors and these changes modify the colours to meet the required contrast ratio 4.5:1.